### PR TITLE
Fix dependabot issues

### PR DESCRIPTION
**Pull request by ksamoray**
Bump the following modules to resolve dependabot errors:
- github.com/Masterminds/goutils v1.1.1
- github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2

Signed-off-by: Kobi Samoray <ksamoray@vmware.com>

### CODE REVIEW

**Review by timdengyun — DISMISSED** [review]
LGTM, Thanks Kobi

**Comment by timdengyun** [comment]
@ksamoray, could you help take a look at go.mod and go.sum conflicts in order to merge the patch.

**Comment by ksamoray** [comment]
> @ksamoray, could you help take a look at go.mod and go.sum conflicts in order to merge the patch. Thanks

@timdengyun as the security alerts are gone, do we still need this? I didn't dismiss these...

**Comment by timdengyun** [comment]
> > @ksamoray, could you help take a look at go.mod and go.sum conflicts in order to merge the patch. Thanks
> 
> @timdengyun as the security alerts are gone, do we still need this? I didn't dismiss these...

I feel that updgrae Golang version is necessary, so we can include this patch as well. @salv-orlando  how is your opinion?

File: go.mod [hunk 1]
```diff
@@ -21,14 +20,81 @@ require (
 	k8s.io/api v0.18.2
 	k8s.io/apimachinery v0.18.3
 	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/kube-openapi v0.0.0-20200410145947-61e04a5be9a6 // indirect
-	k8s.io/kube-proxy v0.18.3 // indirect
 	k8s.io/kubectl v0.17.4
 	sigs.k8s.io/controller-runtime v0.5.2
 )
 
+require (
```
Comment (timdengyun) [inline go.mod:27]: Do we run "go mod tidy" to remove unnecessary modules, it looks like there are many module changes.

**Review by timdengyun — APPROVED** [review]

**Pull request merged**